tests/test_sensor.py

```diff
@@ -208,6 +210,19 @@ async def test_fps_sensor(hass: HomeAssistant) -> None:
     assert entity_state.state == "unknown"
 
 
+async def test_status_sensor(hass: HomeAssistant) -> None:
+    """Test FrigateStatusSensor state."""
+
+    client = create_mock_frigate_client()
+    await setup_mock_frigate_config_entry(hass, client=client)
+    await enable_and_load_entity(hass, client, TEST_SENSOR_FRIGATE_STATUS_ENTITY_ID)
+
+    entity_state = hass.states.get(TEST_SENSOR_FRIGATE_STATUS_ENTITY_ID)
+    assert entity_state
+    assert entity_state.state == "running"
+    assert entity_state.attributes["icon"] == ICON_SERVER
+
+
```

Comment:
Recommend testing at least the error condition as well, by throwing an exception in the API and make it shows up correctly. This is probably the main reason people would want to use a sensor like this, so should test that rare but important case.

Reply:
Is there an example where an exception is manually thrown?

Reply:
This may provide inspiration, it's basically asking the mock API object to raise an exception, then in your test you make sure the status ends up set correctly.

https://github.com/blakeblackshear/frigate-hass-integration/blob/master/tests/test_media_source.py#L961

Reply:
Awesome that is really helpful